services/storage/sqs_test.rb

```diff
@@ -0,0 +1,40 @@
+
+
+require 'aws-sdk-sqs'
+
+
+
```

Comment:
This should be converted into a spec with the code pretty much unchanged so that it tests localstack independently of the cloud layer abstraction. kinda like the sftp spec above tests the sftp service is running.
We should put a meta value on both sftp and this spec so they are by default skipped when running rspec, but can be invoked manually so if a developer wants to test, eg they just want to know the service is working correctly, then they can run the spec w/out having to try to remember all the details and/or invoke sftp manually